apps/obs-web/src/views/DebrieferSelectComp.vue

```diff
@@ -42,27 +36,23 @@ export default createComponent({
 
   setup(props, context) {
     const options: any = ref([]);
+    const valueHolder: any = ref('');
+    const filteredOptions: any = ref([]);
 
     const watcherOptions: WatchOptions = {
       immediate: true
     };
 
-    const valueHolder = computed({
-      get: () => {
-        return props.val;
-      },
-      set: (value: any) => {
-        if (!props.multiple) {
-          value = value ? [value] : [];
-        }
-        context.emit('update:val', value);
-      }
-    });
-
     watch(() => props.lookupQueryOptions, populateLookups, watcherOptions);
 
-    function limitText(count: number) {
-      return 'and ' + count + ' more';
+    function filterFn(val: any, update: any) {
+      update(() => {
+        const needle = val.toLowerCase();
```

Comment:
looks to me like needle is declared but never used.  did you mean to use it in the includes() in the return statement?

Reply:
good catch yes, just added it to the includes statement.